app/services/LoginService.py

```diff
@@ -0,0 +1,20 @@
+import pymysql.cursors
+
+class LoginService(object):
+
+  def __init__(self, connection):
+    self.connection = connection
+
+
+  def login(self, user) -> bool:
+    with self.connection.cursor() as cursor:
+      # Read a single record
+      sql = "SELECT `username`, `password` FROM `User`"
+      cursor.execute(sql)
```

Comment:
This sql query needs to be changed to specifically try to find the user.

Reply:
What do you mean by that? It's getting the data from the User table

Reply:
Like that gets all of the users from the table but it should be more specific so we can use it in phase three.
 Something like
sql = "select `username, password from user where username=username and password = password

Reply:
Oh ok, I was checking it in python but that looks way better